pkg/approver/aws/approver.go

```diff
@@ -0,0 +1,139 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/juju/ratelimit"
+
+	certificates "k8s.io/api/certificates/v1beta1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
+	certificatesv1b1informers "k8s.io/client-go/informers/certificates/v1beta1"
```

Comment:
nit: just `certificatesinformers` since you called the other package `certificates` above

Reply:
done